code/game/objects/items/devices/taperecorder.dm

```diff
@@ -30,10 +36,23 @@
 		mytape = new starting_tape_type(src)
 	update_icon()
 
+/obj/item/taperecorder/proc/readout()
+	var/readout = "<span class='notice'><b>NO TAPE INSERTED</b></span>"
+	if(mytape)
+		if(playing)
+			readout = "<span class='notice'><b>PLAYING</b></span>" //this should just tell you the time it's at but I can't figure it out
+		else
+			var/time = mytape.used_capacity
+			var/mins = round(time / 60)
+			var/secs = time - mins * 60
+			readout = "<span class='notice'><b>[mins]</b>m <b>[secs]</b>s</span>"
+	return readout
```

Comment:
Early returns here? is there any point in  storing the text in readout if you're not going to do anything with it?

Reply:
Well it returns readout, and the proc is embedded in the description of the tape recorder under examine().

Reply:
Why not just return the text itself? seems like it could make things a little cleaner

Reply:
Done